azure-pipelines.yml

```diff
@@ -1,17 +1,12 @@
-# Strings beginning with * seem to precipitate a syntax error...
 trigger:
   branches:
     include:
-    - master
-    - '*_maintenance'
+    - '*'
   tags:
     include:
     - '*'
 pr:
-  autoCancel: true
```

Comment:
Don't we want `autoCancel` on?

Reply:
It's true by default, I can add it back in if you like

Reply:
Oh, cool. That's great then - I've merged.